scripts/build_from_source.sh

```diff
@@ -18,20 +18,22 @@
 
 set -e
 if [ ! -f scripts/build_from_source.sh ];then
-    echo "This script must execute in project root"
+    echo "This script must be executed from project root."
     exit 1
 fi
 
-echo "Build Weex SDK From Source...."
+echo "Build Weex SDK from source...."
 
 npm install
 npm run build:jsfm
 npm run build:polyfill
+npm run build:rax
 
 echo "Weex JS Framework build completed."
 sleep 2
 
-mkdir pre-build
+# Copy built JS resources to their destination.
+[ -d pre-build ] || mkdir pre-build
 cp dist/weex-js-framework.min.js pre-build/native-bundle-main.js
 cp dist/weex-js-framework.min.js android_sdk/assets/native-bundle-main.js
```

Comment:
I think the file structure had been refactored, here should be `android/sdk/assets` instead of `android_sdk/assets`.

Reply:
Got it! I mixed up those folders.